task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -121,27 +121,32 @@ spec:
         # copy content of conffolder to confdir - will be used in next task - related image check
         cp -r .$conffolder/* $(workspaces.workspace.path)/hacbs/$(context.task.name)/confdir
 
-        # We have totally 4 checks here currently
-        check_num=4
+        EXTRACT_DIR="/extracted_base_img"
+        mkdir "${EXTRACT_DIR}"
+        oc image extract ${BASE_IMAGE} "${EXTRACT_DIR}"
```

Comment:
There is that thing, that checks cannot fail, this script may exit with 1 when image extract fail, we should make sure to fetch exit code and then fail using TEST_OUTPUT with ERROR result (exit 0 in bash after)

Reply:
https://github.com/redhat-appstudio/book/blob/main/ADR/0013-integration-service-api-contracts.md